src/ResourceManagement/ServerManagement/ServerManagement/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+ 
+set autoRestVersion=0.16.0-Nightly20160329
+set specUrl="https://raw.githubusercontent.com/fearthecowboy/azure-rest-api-specs/master/arm-servermanagement/2015-07-01-preview/servermanagement.json"
+set source=-Source https://www.myget.org/F/autorest/api/v2
+
+set repoRoot=%~dp0..\..\..\..
+set autoRestExe=%repoRoot%\packages\autorest.%autoRestVersion%\tools\AutoRest.exe
```

Comment:
Any reason why you opt out from using default generator.cmd? e.g. https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/generate.cmd